README.md

```diff
@@ -1,56 +1,87 @@
+of Prometheus in a Kubernetes cluster along with an additional 
+[prometheus-configurer](https://github.com/facebookarchive/prometheus-configmanager) container, 
+providing an HTTP API for managing Prometheus's alerting rules.<br>
+An [alerting service](https://charmhub.io/alertmanager-k8s)
 for Prometheus is offered through a separate Charm.
 
-
 ## Usage
 
-The Prometheus Operator may be deployed using the Juju command line as
+### Deployment
+
+The Prometheus Operator with Prometheus Configurer may be deployed using the Juju command line as
 in
 
 ```sh
-$ juju deploy prometheus-k8s
+$ juju deploy prometheus-with-configurer
 ```
 
-By default the Prometheus Operator monitors itself, but it also
-accepts additional scrape targets over Juju relations with charms that
-support the `prometheus_scrape` interface and preferably use the
-Prometheus charm library. This [charm library](lib/charms/prometheus_k8s/v0/prometheus_scrape.py)
-provides an `add_endpoint()` method that creates additional scrape
-targets. Each scrape target is expected to expose a `/metrics` HTTP
-path that exposes its metrics in a Prometheus compatible format. For
-example, the
-[kube-state-metrics](https://charmhub.io/kube-state-metrics) charm
-integrates with the Prometheus K8S charm in a way that allows you
-import metrics about resources in a Kubernetes cluster by doing:
+### Additional scrape targets
+
+By default the Prometheus Operator monitors itself, but it also accepts additional scrape targets 
+over Juju relations with charms that support the `prometheus_scrape` interface and preferably use 
+the Prometheus charm library. This 
+[charm library](lib/charms/prometheus_k8s/v0/prometheus_scrape.py) provides an `add_endpoint()` 
+method that creates additional scrape targets. Each scrape target is expected to expose 
+a `/metrics` HTTP path that exposes its metrics in a Prometheus compatible format. For example, the
+[kube-state-metrics](https://charmhub.io/kube-state-metrics) charm integrates with the 
+Prometheus k8s charm in a way that allows you import metrics about resources in a Kubernetes 
+cluster by doing:
 
 ```sh
 $ juju deploy kube-state-metrics
-$ juju relate kube-state-metrics prometheus-k8s
+$ juju relate kube-state-metrics prometheus-with-configurer
 ```
 
-In a similar manner any charm that exposes a scrape target may be
-related to the Prometheus charm.
-
-At present it is expected that all relations the Prometheus Operator
-partakes in are within the same Juju model.
-[Further development](https://github.com/canonical/prometheus-operator/issues/58)
-may extend this to allow cross model scrape targets.
+In a similar manner any charm that exposes a scrape target may be related to the Prometheus charm.
+
+At present it is expected that all relations the Prometheus Operator partakes in are within the 
+same Juju model.
+[Further development](https://github.com/canonical/prometheus-operator/issues/58) may extend this 
```

Comment:
The referenced issue is closed, and there does not seem to be any resolution.

Reply:
Nice catch.
This can actually be removed, because cross-model relations already work for COS.